stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -541,10 +541,10 @@ function exp!(A::StridedMatrix{T}) where T<:BlasFloat
         A2 = A * A
         A4 = A2 * A2
         A6 = A2 * A4
-        U  = A * (A6 * (CC[14]*A6 + CC[12]*A4 + CC[10]*A2) +
-                  CC[8]*A6 + CC[6]*A4 + CC[4]*A2 + CC[2]*Inn)
-        V  = A6 * (CC[13]*A6 + CC[11]*A4 + CC[9]*A2) +
-                   CC[7]*A6 + CC[5]*A4 + CC[3]*A2 + CC[1]*Inn
+        U  = A * (A6 * (CC[14]*A6 .+ CC[12]*A4 .+ CC[10]*A2) .+
```

Comment:
`CC[14]*A6` etcetera should be `CC[14].*A6` in order to fuse.    Probably cleaner to just do `@.(CC[14]*A6 + CC[12]*A4 + …)`

Reply:
(I'm not sure why `CC` is an array rather than a bunch of scalars.   Not that any of these micro-optimizations matter for large arrays.)

Reply:
That makes the code three times slower so I rather not do that.

Reply:
Like, I put a MWE on the top so we can talk about actual performance instead of theoretical performance :P

Reply:
That looks like #28126, particularly since when I tried it I saw a 2x gain.  :-\

```julia
julia> @btime f_dots2($A, $A2, $A4, $A6, $CC, $Inn)
  254.430 ns (5 allocations: 560 bytes)
2×2 Array{Float64,2}:
 0.516647  0.311652
 0.186972  0.116884
```

Reply:
Ok, I also get that, but not when using the macro... Did I mess up?

Reply:
Ah nvm, I see now

Reply:
Anyway, just pushed it with explicit dots.

Reply:
Ah, the difference appears to be in the n-ary operations.  The parser doesn't use special n-ary parsing for `.+` but the `@.` macro does.